### DISCUSSION TITLE
NFT fractionalisation

### DISCUSSION THREAD

**Discussion by petushka1** [XRPLF/XRPL-Standards#67] [XLS Ideas (pre standard proposal)]
Hello.
I personally and my Fixarta team are focused on physical collectibles and digital concept designs. After reviewing a proposal and completing own research I’d suggest looking into fractionalisation. Talking about erc-721 there are third party solutions built to split this on a certain amount of erc-20 tokens bound to a specific nft.

While the standard is being designed and developed the team could implement more innovative solution in comparison to Eth. 

This could be addressed through a specific flag set for NFTokenOffer “divided” set by current owner. NFT price would be set as total while each specific fraction price will be dependant on active buy offers. Fractions distribution will be completed once the max fractions number = total buy offers or offer time expired. 

On other hand this mechanism can be built similarly to erc-20 where a number of fungible tokens minted for a specific NFT.

I can clearly see a use case starting from governance (smm tool for collaborative sm accounts (quick fresh idea :)), DeFi, Games and even physical products (will not share at the moment as far as it’s our point of interest ;)).

**Comment by jnr101** [comment]
Have you also considered the erc-1155 standard? I think it offers fractionalization, but also the possibility to assign a specific amount of a certain item. It basically is best of both worlds, both ERC20 (fungible) and ERC721 (non-fungible).

Trying to describe the erc-1155 funcionality in non-ethereum terms, you will be able to assign a unique element (e.g. ipfs uri) to a fungible token (or in XRPL terms, an IOU).

In games you could e.g. use it to assign an amount of gold, or if you would fractionalize to the smallest number, you could e.g. create 10 axes. The axes are interchangeable having the same (fungible) value, while the axe as a type of item is unique.
Also in business these type of tokens can have benefits, like e.g. tokenizing a shipload of grain. (fungible because all the grain in the shipload is the same, yet non-fungible as the shipload is unique in a certain quality and origin of the grain)

For sake of argument, the amount of a token can be described ERC-721 like, it would then be part of the off-chain data. But using it on-chain would require some sort of oracle to inject this data on-chain. I think 'amount' is a trivial property that in quite some cases you would want to have on-chain. erc-1155-alike would place the amount on-chain
